packs/data/spell-effects.db/spell-effect-elephant-form.json

```diff
@@ -85,6 +85,33 @@
                 },
                 "value": {
                     "brackets": [
+                        {
+                            "end": 4,
+                            "start": 4,
+                            "value": {
+                                "size": "lg",
+                                "skills": {
+                                    "ath": {
+                                        "modifier": 18
+                                    }
+                                },
+                                "strikes": {
+                                    "foot": {
+                                        "damage": {
+                                            "modifier": 9
+                                        },
+                                        "modifier": 16
+                                    },
+                                    "tusk": {
+                                        "damage": {
+                                            "modifier": 9
+                                        },
+                                        "modifier": 16
+                                    }
+                                },
+                                "tempHP": 15
+                            }
+                        },
```

Comment:
This should be included in the base override

Reply:
I was afraid something like this might come up. Guess it's time to delve into Battleforms. 😅

Reply:
The properties are named poorly (my fault), but "overrides" contains the base features of the form, and "values" has the bracketed scaling effects.

Reply:
Done (I believe correctly).